flow/project.py

```diff
@@ -1226,6 +1225,19 @@ def print_status(self, jobs=None, overview=True, overview_max_lines=None,
             User provided Jinja2 template file.
         :type template:
             str
+        :param profile:
+            show profile result.
+        :type profile:
+            bool
+        :param eligible_jobs_max_lines:
+            Limit the number of operations and its eligible job count printed in the overview.
+        :type eligible_jobs_max_lines:
+            int
+        :param output_format:
+            status output format, currently supports:
+            'terminal'(default), 'markdown'/'md' and 'html'
```

Comment:
Do we need to support "markdown" and "md"? I would just keep "markdown" if that's code we control (don't change this if it's in mistune).
```suggestion
            'terminal' (default), 'markdown'/'md' or 'html'.
```